StreamingASR/README.md

```diff
@@ -0,0 +1,76 @@
+# Streaming Speech Recognition on Android with Emformer-RNNT-based Model
+
+## Introduction
+
+In the Speech Recognition Android [demo app](https://github.com/pytorch/android-demo-app/tree/master/SpeechRecognition), we showed how to use the [wav2vec 2.0](https://github.com/pytorch/fairseq/tree/master/examples/wav2vec) model on an Android demo app to perform non-continuous speech recognition. Here we're going one step further, using a torchaudio [Emformer-RNNT-based ASR](https://pytorch.org/audio/main/prototype.pipelines.html#torchaudio.prototype.pipelines.EMFORMER_RNNT_BASE_LIBRISPEECH) model in Android to perform streaming speech recognition.
+
+## Prerequisites
+
+* PyTorch 1.10.1 and torchaudio 0.10.1 or above (Optional)
+* Python 3.8 (Optional)
+* Android Pytorch library org.pytorch:pytorch_android_lite:1.10.0
+* Android Studio 4.0.1 or later
+
+## Quick Start
+
+### 1. Get the Repo
+
+Simply run the commands below:
+
+```
+git clone https://github.com/pytorch/android-demo-app
+cd android-demo-app/StreamingASR
+```
+
+If you don't have PyTorch 1.10.1 and torchaudio 0.10.1 installed or want to have a quick try of the demo app, you can download the optimized scripted model file [streaming_asr.ptl](https://drive.google.com/file/d/1awT_1S6H5IXSOOqpFLmpeg0B-kQVWG2y/view?usp=sharing), then drag and drop it to the `StreamingASR/app/src/main/assets` folder inside `android-demo-app/StreamingASR`, and continue to Step 3.
+
+Also you need to download [Eigen](https://eigen.tuxfamily.org/), a C++ template library for linear algebra, for Android NDK build required to run the app (see last section of this README for more info):
+```
+mkdir external; cd external
+git clone https://github.com/jeffxtang/eigen
```

Comment:
Is it some fork of eigen with some changes on the top? 
Do you know if official eigen repository exists on github?

Reply:
Yes it's based on a fork of Eigen which was created so it can be built on Android with NDK. The original Eigen repo doesn't work on Android. I mentioned both in the README of https://github.com/jeffxtang/eigen, which was created to modify the CMakeLists.txt (changes here https://github.com/jeffxtang/eigen/compare/diff?expand=1) to make the original fork really build successfully on Android (otherwise, I got Android build errors).